### DISCUSSION TITLE
pages setup for Enforce https but google chrome browser showing "not secure"

### DISCUSSION THREAD

**Discussion by abiyug** [community/community#62563] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have the following URL hosted on github pages "https://datarecode.com". A google chrome browser from mac os shows it as "not secure", but not from a google chrome on android phone. No similar warnings on other browsers.  The "Enforce HTTPS" is enabled on pages.  Any support is appreciated.

![image](https://github.com/community/community/assets/12779898/c9c05420-0351-4cc6-9c42-9622fc3f16b2)

**Comment by queenofcorgis** [comment]
Hi there @abiyug - interesting and thanks for checking if Enforce HTTPS was on, have you tried at all in incognito mode or unchecking the box then rechecking it?

**Comment by revoltig4sg** [comment]
The "not secure" warning in Google Chrome on your Mac OS might be due to the lack of an SSL certificate on your GitHub Pages hosted website. By default, GitHub Pages uses the "github.io" domain, which is covered by their SSL certificate. However, if you are using a custom domain like "datarecode.com," you need to configure SSL separately. 
 
To resolve this issue and enable HTTPS for your GitHub Pages website, you can follow these steps: 
 
1. Obtain an SSL certificate: You can obtain an SSL certificate from a trusted certificate authority (CA) like Let's Encrypt or purchase one from a certificate provider. Some hosting providers also offer free SSL certificates. Make sure to choose an SSL certificate that matches your domain name ("datarecode.com"). 
2. Configure SSL for your custom domain: Once you have obtained the SSL certificate, you need to configure it for your custom domain. This typically involves updating the DNS settings for your domain to point to the correct SSL certificate. 
3. Enable HTTPS redirection: After configuring the SSL certificate, you need to enable HTTPS redirection. This ensures that all HTTP requests are automatically redirected to HTTPS. You can typically enable this option in your hosting provider's settings or through a configuration file. 
4. Update GitHub Pages settings: Finally, update the GitHub Pages settings for your repository to use the custom domain with HTTPS. You can do this by adding or updating the "CNAME" file in your repository with your custom domain. 
Once these steps are completed, your GitHub Pages website should be accessible over HTTPS, and the "not secure" warning should no longer appear in Google Chrome on your Mac OS. It's worth noting that it might take some time for the changes to propagate across the internet due to DNS caching. 

I hope this will help you.
